Description: Work around a maven-debian-helper bug where reactor dependencies
 aren't properly recognized and switched to the 'debian' version, thus causing
 a build failure.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/jaxb-ri/runtime/impl/pom.xml
+++ b/jaxb-ri/runtime/impl/pom.xml
@@ -67,6 +67,7 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>jaxb-core</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.jvnet.staxex</groupId>
--- a/jaxb-ri/xjc/pom.xml
+++ b/jaxb-ri/xjc/pom.xml
@@ -73,6 +73,7 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>jaxb-core</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.jmockit</groupId>
@@ -97,6 +98,7 @@
         <dependency>
             <groupId>com.sun.xml.bind.external</groupId>
             <artifactId>rngom</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>com.sun.xml.dtd-parser</groupId>
--- a/jaxb-ri/jxc/pom.xml
+++ b/jaxb-ri/jxc/pom.xml
@@ -70,14 +70,17 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>jaxb-xjc</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>jaxb-core</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>jaxb-runtime</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>com.sun.istack</groupId>
--- a/jaxb-ri/jaxb-xjc-jdk9/pom.xml
+++ b/jaxb-ri/jaxb-xjc-jdk9/pom.xml
@@ -61,6 +61,7 @@
         <dependency>
             <groupId>org.glassfish.jaxb</groupId>
             <artifactId>jaxb-xjc</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
@@ -92,6 +93,7 @@
         <dependency>
             <groupId>com.sun.xml.bind.external</groupId>
             <artifactId>rngom</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>com.sun.istack</groupId>
--- a/jaxb-ri/jaxb-jxc-jdk9/pom.xml
+++ b/jaxb-ri/jaxb-jxc-jdk9/pom.xml
@@ -61,6 +61,7 @@
         <dependency>
             <groupId>org.glassfish.jaxb</groupId>
             <artifactId>jaxb-jxc</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
@@ -72,14 +73,17 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>jaxb-xjc</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>jaxb-core</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>jaxb-runtime</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>com.sun.istack</groupId>
